packages/imperative/src/OverridesLoader.ts

```diff
@@ -110,7 +83,7 @@ export class OverridesLoader {
         displayName,
 
         // zowe cli will always add `Zowe` to it's list of service names
-        service: config?.name === ZOWE_CLI_PACKAGE_NAME ? DefaultCredentialManager.SVC_NAME : config?.name,
+        service: config?.credentialServiceName || (config.name !== ZOWE_CLI_PACKAGE_NAME ? config.name : null),
```

Comment:
I noticed `overrides` was added to `IConfig`, but I couldn't find `credentialServiceName`.
Should it be added?

Reply:
`credentialServiceName` is a property that already existed in the IImperativeConfig interface: https://github.com/zowe/imperative/blob/next/packages/imperative/src/doc/IImperativeConfig.ts#L316